docs: Update encryption key instructions (#1836)
n8n-io/n8n-docs: docs/hosting/environment-variables/configuration-methods.md

n8n creates a random encryption key automatically on the first launch and saves
it in the `~/.n8n` folder. n8n uses that key to encrypt the credentials before
they get saved to the database. If the key is not yet in the settings file,

> it's possible to specify it using an environment variable, so that n8n 

Reviewer: ```suggestion
you can set it using an environment variable, so that n8n 
```

> will use your custom key instead of generating a new one.

Reviewer: ```suggestion
uses your custom key instead of generating a new one.
```

> On queue mode, it is required to specify the encryption key env var for all workers.

Reviewer: ```suggestion
In queue mode, it is required to specify the encryption key environment variable for all workers.
```
